src/njoy21/input/HEATR/Card2.hpp

```diff
@@ -1,9 +1,56 @@
 class Card2 {
+  public:
   using Matd = argument::common::Matd;
```

Comment:
How did you make a comment with a thumbs up? (I feel like an old man asking a teenager how to use SnapChat.